design/sd-local.md

```diff
@@ -157,6 +157,46 @@ $ sdlocal build [job-name] [options]
 
 ### Configuration
 
+config support multiple Screwdriver cluster.  
+The config file as `$HOME/.sdlocal/config` with YAML format.  
+
+```yaml
+configs:
+  default:
+    api-url: <Screwdriver API URL>
+    store-url: <Screwdriver Store URL>
+    token: <Screwdriver API Token>
+    launcher:
+      version: <Launcher Version>
+      image: <Launcher image name>
+  yourConfigName:
+    api-url: <Screwdriver API URL>
+    store-url: <Screwdriver Store URL>
+    token: <Screwdriver API Token>
+    launcher:
+      version: <Launcher Version>
+      image: <Launcher image name>r
+current: default
+```
+
+- `default` is a special config which are created automatically when the first time the user use sd-local. However, default is not created if the user create a config explicitly with sd-local config create <config-name> in the first time, and, the specified name is used instead of default.
+
+- `current` points to default (or first created config) in the first time
+
+You can create a cluster config with the command below:
+
+```bash
+$ sdlocal config create [config-name]
+```
```

Comment:
I would be better to add that can't create duplicate cluster names.

Reply:
added